frontend/src/App.js

```diff
@@ -1,9 +1,58 @@
-import React from 'react'
+import React, { useState } from 'react'
+
+
+// La till en ny component 
+import { Login } from './Login'
+
+//const URL = 'https://harry-potter-auth.herokuapp.com/users'
+
+const URL = 'http://localhost:8080/users'
```

Comment:
Shouldn't this be using the backend you built?  The site won't work like this.  Please fix @claragyokeres @Angelicabrodin